flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -80,6 +80,9 @@ public class ScanContext implements Serializable {
   private static final ConfigOption<Boolean> INCLUDE_COLUMN_STATS =
       ConfigOptions.key("include-column-stats").booleanType().defaultValue(false);
 
+  private static final ConfigOption<Boolean> PLAN_SINGLE_WHOLE_FILE_PER_TASK =
```

Comment:
So I see you introduced this new conf option to control this feature.

Could you put this conf `PLAN_SINGLE_WHOLE_FILE_PER_TASK` in the PR description? Also, how does this config get passed down to `ScanContext` from client code? is it through table properties?

Reply:
Added to the PR description. The ScanContext will be created by the client code using the builder. Woops, missed adding  the corresponding  IcebergSource builder API. will update the PR.